### PR TITLE
Added option to CRUD the state of the print form (values)

### PR DESCRIPTION
**Pull request by weskamm**
Adds a window in the printform to save / load / delete the form values for faster printing

### CODE REVIEW

File: app/view/form/PrintController.js [hunk 1]
```diff
@@ -576,5 +577,21 @@ Ext.define('MoMo.client.view.form.PrintController', {
...
+
+    saveLoadSettings: function() {
+        var me = this;
+        var viewModel = me.getView().getViewModel();
+        Ext.create('Ext.window.Window', {
+            width: 800,
+            height: 350,
+            modal: true,
+            bind: {
+                title: viewModel.get('saveLoadSettings')
```
Comment (dnlkoch) [inline app/view/form/PrintController.js:590]: Or just `{saveLoadSettings}`?

File: app/view/grid/PrintFormSettingsController.js [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        var me = this;
+        var view = me.getView();
+
+        new Ext.util.KeyMap({
+            target: view.getEl(),
+            binding: [{
+                key: 'c',
+                ctrl: true,
+                fn: function() {
+                    me.copyUrlToClipboard();
```
Comment (dnlkoch) [inline app/view/grid/PrintFormSettingsController.js]: Could not find this method in this changeset, not implemented (yet)?

**Review by dnlkoch — COMMENTED** [review]
LGTM!

File: app/view/form/PrintController.js (hunk 1 above)
Comment (weskamm) [reply app/view/form/PrintController.js:590]: tried it, didnt work, will keep it this way

**Comment by weskamm** [comment]
Thanks for review, merging now

**Pull request merged**